lib/internal/test_runner/mock.js

```diff
@@ -202,6 +202,64 @@ class MockTracker {
     return mock;
   }
 
+  getter(
+    object,
+    methodName,
+    implementation = kDefaultFunction,
+    options = kEmptyObject
+  ) {
+    if (implementation !== null && typeof implementation === 'object') {
+      options = implementation;
+      implementation = kDefaultFunction;
+    }
+
+    validateObject(options, 'options');
```

Comment:
We can skip the validation if we know it's an object already.
```suggestion
    } else {
      validateObject(options, 'options');
    }
```